test/e2e/common/setup-kubestellar.sh

```diff
@@ -83,6 +91,20 @@ SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
 COMMON_SRCS="${SRC_DIR}/../common"
 source "$COMMON_SRCS/setup-shell.sh"
 
+# Get scenario configuration
+scenario_config=($(get_scenario_config "$TEST_SCENARIO"))
+if [ $? -ne 0 ]; then
+    echo "Invalid scenario: $TEST_SCENARIO" >&2
+    list_scenarios
+    exit 1
+fi
+
+# Parse scenario configuration
+declare -A config
+for i in "${!scenario_config[@]}"; do
+    config[$i]="${scenario_config[$i]}"
+done
```

Comment:
Why have two copies of the same associative array? Why not just have the code below consume the first copy?